### ISSUE TITLE
This component affects the chatbot interface from streamlit

### ISSUE THREAD

**Issue by Odrec** [gabrieltempass/streamlit-navigation-bar#3]
### Checklist

- [X] I have searched the [existing issues](https://github.com/gabrieltempass/streamlit-navigation-bar/issues) for similar issues.
- [X] I added a very descriptive title to this issue.
- [X] I have provided sufficient information below to help reproduce this issue.

### Summary

If you use this component along the streamlit chat interface, it affects the position of the input for the chat and also the automatic scrolling when the chat gets bigger than the window. This is with chat streaming. This is a sceenshot with the component and the chat used in the same app:

![Screenshot from 2024-04-16 15-39-45](https://github.com/gabrieltempass/streamlit-navigation-bar/assets/9385209/4f257ea3-ca41-4cff-b43b-784dd0c4993b)

Notice how the message disappears behind the input and I can't scroll down to show the complete message.

This is the result without this component which is how it is supposed to look:

![Screenshot from 2024-04-16 15-44-46](https://github.com/gabrieltempass/streamlit-navigation-bar/assets/9385209/9f0eb88d-5a54-4a00-a4ec-74e9e996cca5)

In this screenshot the input for the chat is a bit higher in the window and the entire chat results scrolled to the bottom automatically so I can see it all unlike with what happens when using this component.

### Reproducible code example

```
import streamlit as st
from streamlit_navigation_bar import st_navbar

st.set_page_config(page_title="test", layout="wide", page_icon="❗")


def display_pages_tabs(default_id):
    pages = ["Chatbot 🤖", "Chat with Documents 📖"]

    st_navbar(pages, selected=pages[default_id])


display_pages_tabs(0)

user_message = "Hello!" * 10000
bot_message = "Goodbye!" * 10000

st.chat_message("user").write(user_message)

st.chat_message("assistant").write(bot_message+"END")

st.chat_input("Whatever")
```

### Steps to reproduce

Just run the above code snippet.

The word "END" and the end of the assistant message won't be visible and you can't scroll to see it.

If you comment out the `display_pages_tabs(0)` line, then you can see the entire message.

### Expected behavior

That the entire message is visible and that you can normally scroll on the chat interface.

### Is this a regression?

- [ ] Yes, this used to work in a previous version.

### Debug info

- Streamlit version: latest
- Streamlit Navigation Bar version: latest
- Python version: 3.11
- Operating System: Linux
- Browser: Brave

**Comment by gabrieltempass** [comment]
Hey @Odrec, thanks for opening the bug report! I will take a look at this as soon as possible.

**Comment by gabrieltempass** [comment]
@Odrec, I released version 3.1.2 that fixes the issues you mentioned. You can update the package with `pip install --upgrade streamlit-navigation-bar`, and `st.chat_input` should work as expected.

If you notice anything else, please let me know!

**Comment by Odrec** [comment]
> @Odrec, I released version 3.1.2 that fixes the issues you mentioned. You can update the package with `pip install --upgrade streamlit-navigation-bar`, and `st.chat_input` should work as expected.
> 
> If you notice anything else, please let me know!

Great thx!
